### ISSUE TITLE
New version overlay crashes

### ISSUE THREAD

**Issue by HankChen5408** [TekkenOverlay/TekkenOverlay#167]
Why do I still crash? Can everyone run it?

**Comment by haziqtheunique** [comment]
Can confirm, New update still crashes at launch.

**Comment by Paromc** [comment]
crash niau

**Comment by Puppydup** [comment]
It works for me, rtx 3070 ryzen 5700x3d

**Comment by CCauldron** [comment]
Are you boys on AMD GPUs? If yes rollback to driver version 23.12.1. (or prior) I haven't crashed since doing that if that helps to solve your issue.

**Comment by haziqtheunique** [comment]
> Are you boys on AMD GPUs? If yes rollback to driver version 23.12.1. (or prior) I haven't crashed since doing that if that helps to solve your issue.

Nope. I'm on an RTX 3090 & I crash every time I'm on the title screen after loading the overlay prior to launching the game.

**Comment by lebelchris** [comment]
> > Are you boys on AMD GPUs? If yes rollback to driver version 23.12.1. (or prior) I haven't crashed since doing that if that helps to solve your issue.
> 
> Nope. I'm on an RTX 3090 & I crash every time I'm on the title screen after loading the overlay prior to launching the game.

same crash spot for me. I can see kazuya and the overlay text but doing anything crashes

**Comment by HankChen5408** [comment]
https://github.com/TekkenOverlay/TekkenOverlay/issues/169

This version solved my problem, try it
